Assets/Scripts/OrchestraArmy/Room/Room.cs

```diff
@@ -87,6 +100,7 @@ void CreateFloors()
                 foreach (Walker myWalker in _walkers)
                 {
                     Grid[(int)myWalker.Pos.x, (int)myWalker.Pos.y] = GridSpace.Floor;
+                    _floors.Add(new Vector2((int)myWalker.Pos.x, (int)myWalker.Pos.y)); //add position to floor list
```

Comment:
pas op er kunnen hier dubbele locaties in de _floors vector komen
(walkers kunnen over floors lopen)